nengo_viz/static/viz_slider.js

```diff
@@ -212,14 +212,44 @@ VIZ.Slider.prototype.generate_menu = function() {
     var self = this;
     var items = [];
     items.push(['set range', function() {self.set_range();}]);
+    items.push(['set value', function() {self.user_value();}]);
 
     // add the parent's menu items to this
     // TODO: is this really the best way to call the parent's generate_menu()?
     return $.merge(items, VIZ.Component.prototype.generate_menu.call(this));
 };
 
+VIZ.Slider.prototype.user_value = function () {
+    var slider_index = 0
+    if (this.sliders.length > 1) {
+        do{
+            slider_index = prompt('Enter the index of the slider to Change');
+            if (new_value == null) {
+                return;
+            }
+        } while (!(VIZ.is_num(slider_index)));
```

Comment:
If the user inputs an invalid number, it should not keep asking the user until they give a valid answer.